pkg/manifests/manifests.go

```diff
@@ -222,8 +222,7 @@ var (
 	ThanosQuerierServiceMonitor         = "thanos-querier/service-monitor.yaml"
 	ThanosQuerierPrometheusRule         = "thanos-querier/prometheus-rule.yaml"
 	ThanosQuerierRoute                  = "thanos-querier/route.yaml"
-	ThanosQuerierOauthCookieSecret      = "thanos-querier/oauth-cookie-secret.yaml"
```

Comment:
CMO needs to remove the associated objects so that we don't leave anything behind after upgrading from 4.9 to 4.10.